CONTRIBUTION.md

```diff
@@ -0,0 +1,161 @@
+nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+### Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+### Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at [http://contributor-covenant.org/version/1/4][version]
+
+[homepage]: http://contributor-covenant.org
+[version]: http://contributor-covenant.org/version/1/4/
+
+## Pull Request Process 
+
+1. Eclipse Che documentation project follows forking workflow. To learn more, read [Atlassian Git Tutorial, Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow). Before you start working, fork the repository, make sure you create all branches in your own fork.
+2. `master`  is the default working branch. Unless your change is intended for a specific version, send the PR against a `master`. See the Release branches to learn more.
+3. Specify as much information as possible in the PR template. 
+
+### Release branches
+[WIP]
+
+## Creating documentation
+
+### AsciiDoc Formatting
+Eclipse Che documentation use AsciiDoc for formatting. See [AsciiDoc Writer's Guide](https://asciidoctor.org/docs/asciidoc-writers-guide/) for syntax and general help with AsciiDoc.
+
+#### Using `test-adoc` for AsciiDoc syntax verification
+To verify that the syntax of your document is correct, use [`test-adoc`](https://github.com/jhradilek/check-links) script. 
+`test-adoc.sh` is integrated into the `run.sh` script in the root directory of the project.
+ 
+To run `test-adoc`, execute from the root directory of the project: 
+```
+$ bash run.sh -test-adoc <PATH_TO_THE_FILE>
+```
+
+### Documentation Structure
+Eclipse Che documentation follow [Modular Documentation initiative](https://redhat-documentation.github.io/modular-docs/). Use templates provided in the [Modular Documentation Reference](https://redhat-documentation.github.io/modular-docs/#creating-modules) to create new user stories.
```

Comment:
"The Eclipse Che documentation project follows guidelines from the Modular Documentation initiative." (Also, is modular documentation actually capitalized? I'm not sure.)